src/GoTrueClient.ts

```diff
@@ -417,7 +417,7 @@ export default class GoTrueClient {
       })
       if (error || !data) return { data: null, user: null, session: null, error }
 
-      if (data?.user?.confirmed_at) {
+      if (data?.user?.email_confirmed_at) {
```

Comment:
this looks like a breaking change 

We will need to support old versions of GoTrue as well right?

So `data?.user?.confirmed_at || data?.user?.email_confirmed_at`

Side note - we didnt remove `confirmed_at` from GoTrue right?

Reply:
do we need to include `phone_confirmed_at`?